config/matchmaker_config.yaml

```diff
@@ -25,8 +25,10 @@ api:
     port: 50502
     proxyport: 51502
   
-evalutor: 
-  interval: 10
```

Comment:
Will this break existing examples? Also "evaluator"

Reply:
Nope. :) It was already misspelt and already kinda broken :')  I ve added it back and now I think the existing example evaluator will behave different :p thats gonna be deprecated though in the next couple PRs.